index.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+@include_once __DIR__ . '/vendor/autoload.php';
+
+use function Sentry\init;
+use function Sentry\captureException;
+use function Sentry\configureScope;
+use Sentry\State\Scope;
+
+Kirby::plugin("thathoff/sentry", [
+    'config' => [
+        'dsn' => null,
+        'environment' => 'production',
+        'addUserContext' => true,
+    ],
+    'hooks' => [
+        'system.exception' => function ($exception) {
+            // when no dsn is configured we do nothing
+            if (!$dsn = option('thathoff.sentry.dsn')) {
```

Comment:
sollte es nicht `!($dsn = option('thathoff.sentry.dsn'))` sein? PHP 7.4 wirft mir einen Fehler aus: `The assignment expression is not valid. The input to an assignment operator must be an object that is able to accept assignments, such as a variable or a property`